combat-trainer.lic

```diff
@@ -172,6 +172,12 @@ class SetupProcess
     else
       @equipment_manager.wield_weapon(game_state.weapon_name, game_state.weapon_skill)
     end
+
+    # Invoke Focus
+    if game_state.weapon_skill == 'Targeted Magic'
+      bput("invoke #{game_state.weapon_name}", 'You')
```

Comment:
If you did something like `bput(...) if game_state.weapon_name` I think this would also support syntax like:
```
weapon_training:
  Targeted Magic:
```

I.e. Train TM without weapons, which some people have asked for.